packages/component-library-angular/src/checkbox/Checkbox.spec.ts

```diff
@@ -0,0 +1,396 @@
+import { render, screen } from '@testing-library/angular';
+import '@testing-library/jest-dom';
+import { clearElements } from '../utils';
+import { UtrechtCheckboxAttr } from './component';
+
+afterEach(() => {
+  clearElements();
+});
+
+describe('Checkbox', () => {
+  it('renders a checkbox role element', async () => {
+    await render('<input type="checkbox" utrecht-checkbox />', {
+      declarations: [UtrechtCheckboxAttr],
+    });
+
+    const checkbox = screen.getByRole('checkbox');
+
+    expect(checkbox).toBeInTheDocument();
+    expect(checkbox).toBeVisible();
+  });
+
+  it('renders an HTML input type=checkbox element', async () => {
+    const { fixture } = await render('<input type="checkbox" utrecht-checkbox />', {
+      declarations: [UtrechtCheckboxAttr],
+    });
```

Comment:
There should at least be `By.query('input[type="checkbox"]')` to run the test.

Reply:
I'm gonna use  `container.querySelector('input[type="checkbox"]')`